raft/log.go

```diff
@@ -991,12 +986,12 @@ func (l *Log) applier(done chan chan struct{}) {
 	}
 }
 
-// apply a log initialization command by parsing and setting the configuration.
-func (l *Log) applyInitialize(e *LogEntry) error {
+// mustApplyInitialize a log initialization command by parsing and setting the configuration.
+func (l *Log) mustApplyInitialize(e *LogEntry) {
 	// Parse the configuration from the log entry.
 	config := &Config{}
 	if err := NewConfigDecoder(bytes.NewReader(e.Data)).Decode(config); err != nil {
-		return fmt.Errorf("initialize: %s", err)
+		panic("decode: " + err.Error())
```

Comment:
We're going to leave these panics in for production code? Because it's such a bad failure, we should just go down?